shell/scripts/extension/parse-tag-name

```diff
@@ -6,10 +6,10 @@ GITHUB_WORKFLOW_TYPE=$3
 
 # Ensure "catalog" workflow release tag name does not match a pkg/<pkg-name>
 if [[ "${GITHUB_WORKFLOW_TYPE}" == "catalog" ]]; then
-  for d in pkg/*/ ; do
+  for d in pkg/*/; do
```

Comment:
the deleted whitespace is result of [shell-format](https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format) run